scopus-import/src/test/java/no/sikt/nva/scopus/utils/ScopusGenerator.java

```diff
@@ -279,6 +280,21 @@ private HeadTp randomHeadTpWithContributorsFromAuthorTp(AuthorTp authorTp) {
         return head;
     }
 
+    private static @NotNull AffiliationTp randomAffiliationTp() {
+        var organization = randomOrganizationTp();
+        var affiliationTp = new AffiliationTp();
+        affiliationTp.setCountry(randomString());
+        affiliationTp.setAfid(randomString());
+        affiliationTp.getOrganization().add(organization);
+        return affiliationTp;
+    }
+
+    private static @NotNull OrganizationTp randomOrganizationTp() {
```

Comment:
Do we need this Intellij annotation?

Reply:
Possibly not, it was likely auto-format stuff. There are multiple issues in this file, which I have not addressed